src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/AssessmentTest.kt

```diff
@@ -987,7 +987,7 @@ class AssessmentTest : IntegrationTestBase() {
 
             val persistedPlacementRequest = placementRequestRepository.findByApplication(application)!!
 
-            assertThat(persistedPlacementRequest.allocatedToUser.id).isIn(listOf(matcher1.id, matcher2.id))
+            assertThat(persistedPlacementRequest.allocatedToUser!!.id).isIn(listOf(matcher1.id, matcher2.id))
```

Comment:
It might be worth adding:
```
assertThat(persistedPlacementRequest.allocatedToUser).isNotNull
```
as an additional assertion before this one so that if it is null the test fails cleanly instead of through a `NullPointerException`.